sdk/resourcemanager/azure-resourcemanager-appplatform/src/main/java/com/azure/resourcemanager/appplatform/models/SpringConfigurationService.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.resourcemanager.appplatform.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.resourcemanager.appplatform.fluent.models.ConfigurationServiceResourceInner;
+import com.azure.resourcemanager.resources.fluentcore.arm.models.ExternalChildResource;
+import com.azure.resourcemanager.resources.fluentcore.model.HasInnerModel;
+
+import java.util.List;
+
+/** An immutable client-side representation of an Azure Spring Cloud Configuration Service. */
+@Fluent
+public interface SpringConfigurationService
```

Comment:
Any reason why it is named `SpringConfigurationService`? Whiat kind of service it is?

Reply:
It's like `ConfigServer` in Basic/Standard Tier except that itself can be configured in Enterprise Tier(cpu, memory, instances, whether bind apps to it).

Reply:
Or it should be named `TanzuConfigurationService`?

Reply:
OK, current name should be fine. I assume the actual configure of this `SpringConfigurationService` would be in your future PR?

Reply:
The service is configurable but I prefer configuring the defaults for the user and leaving the customization to the future if requested.

Reply:
Sure.